5exercise3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <title>Task</title>
+    
+    </head>
+<body>
+
+    <script>
+let accumulator = new Accumulator(1); // начальное значение 1
+function Accumulator(startingValue) {
+    let value;
+  this.value = startingValue,
+  this.read = function(){
+      let inPut;
+      while (isNaN(inPut) || inPut === '' || inPut === null){
```

Comment:
The formatting makes it almost impossible to read the code because you don't follow the indentation rules. Check this article https://learn.javascript.ru/coding-style. It's better to separate the validation logic and logic related to entering the number.